app/helpers/foreman_ansible/ansible_reports_helper.rb

```diff
@@ -67,6 +69,17 @@ def ansible_report?(log)
       false
     end
 
+    def show_full_error_message_value(message_value)
+      tag.div class: 'replace-hidden-value' do
+        link_to_function(icon_text('plus', '', class: 'small'), 'replace_value_control(this, "div")',
```

Comment:
since this change requires foreman 3.6,
can you update the requires_foreman line in this plugin?
and can you also update the REX dependency in https://github.com/theforeman/foreman_ansible/blob/master/foreman_ansible.gemspec#L23?